src/transformers/image_processing_utils.py

```diff
@@ -75,6 +75,9 @@ class ImageProcessingMixin(PushToHubMixin):
 
     def __init__(self, **kwargs):
         """Set elements of `kwargs` as attributes."""
+        # This key was saved while we still used `XXXFeatureExtractor` for image processing. Now we use
+        # `XXXImageProcessor`, this attribute and its value are misleading.
+        kwargs.pop("feature_extractor_type", None)
```

Comment:
I feel we can also pop `image_processor_type`. We only need this when we save (which is done automatically).
But it doesn't matter much + not sure if any user code uses that attribute 😅

Reply:
Agreed - I'd be pro popping but removing is an easy way to inadvertently break lots of things just before Christmas 🙈